data_science/scripts/toggle_asg.py

```diff
@@ -42,6 +81,14 @@
             desired_size=desired_size
         )
 
+    # Scaling rules
+    elif args['--keep-on'] or args['--allow-scale-down']:
+        if args['--keep-on']:
+            keep_on(asg_client, asg_name)
+        else:
+            allow_scale_down(asg_client, asg_name)
+
```

Comment:
Can these be a single switch and switch between DesiredCapacity 1 or 0?  It's a bit confusing to allow both `keep-on` and `allow-scale-down` on the command line.